kaggle_tools_update.Dockerfile

```diff
@@ -0,0 +1,14 @@
+# This Dockerfile is a temporary solution to unblock the 
+# community team by updating their library latest using
+# the latest python image.
+# This is needed until we resolved the issues with the
+# main build.
+
+# Usage:
+#   docker build --rm -t kaggle/python-build -f kaggle_tools_update.Dockerfile .
+#   ./test
+#   ./push (if tests are passing)
+
+FROM gcr.io/kaggle-images/python:latest
```

Comment:
We can't build new image from master right now. There is different issues that @crawforc3 has been working on fixing but it will likely take some more time. To unblock new Learn content.

The current gcr.io/kaggle-images/python:latest image is the one used by our workers. It has an old version of learntools (the version installed on our last green build on July 10th).

This Dockerfile will simply take that image and run a command to update the learntools to the latest version.